main/WebServer.cpp

```diff
@@ -964,7 +964,7 @@ namespace http {
 				if (ii == HTYPE_RaspberryGPIO)
 					bDoAdd = false;
 #endif
-				if (((ii == HTYPE_1WIRE) && (!C1Wire::Have1WireSystem())) || (ii == HTYPE_PythonPlugin))
+				if (ii == HTYPE_PythonPlugin)
```

Comment:
why is this removed ? (Have1WireSystem)
Now on a (windows) system that does not have 1wire we can add this hardware?